src/lib/comms/Makefile

```diff
@@ -121,12 +124,21 @@ obj-networking-$(MAVLINK)-extra-cflags += \
 obj-networking-$(LWM2M) += \
 	sol-lwm2m.o
 
+obj-bluetooth-$(BLUETOOTH) += \
+	sol-bluetooth.o
+
+obj-bluetooth-$(PLATFORM_LINUX) += \
```

Comment:
Will take a look. I was wondering about making "bluez" when writing this, but as BlueZ is the default Bluetooth stack for Linux, I thought this made sense. But no hard feelings about this.